.buildkite/pipeline.yml

```diff
@@ -37,6 +37,13 @@ steps:
       ./gradlew testDebugUnitTest
     plugins: [$CI_TOOLKIT]
 
+  - label: 'Dependency diff'
+    if: build.pull_request.id != null
```

Comment:
Question (❓): This is a very interesting addition, I like that, should we maybe apply the same on all our projects, wdyt? 🤔

Reply:
But I think this is applied in other projects! Did I miss it somewhere?

Reply:
Ah, interesting, I just checked all JP/WPAndroid, WCAndroid and DOAndroid and indeed this is included in them all, I might be dreaming, apologies for the false alarm, I just probably noticed that just now, just because it was an actual change/addition... 🤷 😊